vnc.html

```diff
@@ -221,6 +221,9 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br />VNC</h1>
                                 <label for="noVNC_setting_repeaterID">Repeater ID:</label>
                                 <input id="noVNC_setting_repeaterID" type="input" value="" />
                             </li>
+                            <li>
+                                <label><input id="noVNC_setting_show_dot" type="checkbox" /> Show Dot when No Cursor</label>
+                            </li>
```

Comment:
I would prefer seeing this placed below the websocket-settings. Either in it's own 'section' with a \<hr\> separating stuff, or  together with the logging-setting.

Reply:
I would prefer it in its own section. Will redo the pull request.

Reply:
Sounds great, thank you!